scripts/vagrant/README.md

```diff
@@ -0,0 +1,68 @@
+Vagrant setup
+===
+
+## Prerequisites
+
+- [Create a Github app](https://github.com/settings/apps) and download the private key from the app's page.
+- Put it under `$PROJECT_ROOT/.secrets/builder-github-app.pem`
+- Create a new file at `$PROJECT_ROOT/.secrets/habitat-env` and write the following into it:
```

Comment:
Maybe call it `builder-env` since the variables mainly affect builder and not the rest of habitat, no?

Also.. could https://github.com/kinvolk/habitat/tree/master/support/bldr.env be used for these?

Reply:
We need to use something which is not in git, as it is user specific configuration.

Details like file name / location can be discussed later, let's focus on getting it to work reliable.